electrum/gui/qt/paytoedit.py

```diff
@@ -58,12 +58,12 @@ class PayToLineError(NamedTuple):
     is_multiline: bool = False
 
         #todo uncomment when QR Read will be work fine
```

Comment:
if its working, remove the comment please

Reply:
done